tester/messagetracker.go

```diff
@@ -0,0 +1,95 @@
+package tester
+
+// MessageTracker tracks message offsets for each topic for convenient
+// 'expect message x to be in topic y' in unit tests
+type MessageTracker struct {
```

Comment:
I would suggest bind the MessageTracker to a topic. That will simplify the interface.